Paco-Server/src/com/google/sampling/experiential/server/JobStatusServlet.java

```diff
@@ -140,7 +140,7 @@ private void writeJobStatus(HttpServletResponse resp, ReportJobStatus jobReport,
     if (!Strings.isNullOrEmpty(errorMessage)) {
       out.append("<th>" + "Error" + "</th>");
       out.append("<td>").append(errorMessage).append("</td>");
-    } else if (!Strings.isNullOrEmpty(jobReport.getLocation())) {
+    } else if (!Strings.isNullOrEmpty(jobReport.getLocation()) && !jobReport.getLocation().equalsIgnoreCase("na")) {
```

Comment:
When does it get set to "na"?

Reply:
Migration Back end jobs, which does not have physical location in cloud storage, uses this default location name (to indicate that job has been completed successfully).

Reply:
Ah, OK, so migration backend jobs do not produce a document in cloud storage therefore the report "na" as the location of the document in the jobstatus record? If so, we should make this a constant in the code and have both those jobs and this test use it.